http/config.go

```diff
@@ -142,3 +148,16 @@ func allowReferer(r *http.Request, cfg *ServerConfig) bool {
 
 	return false
 }
+
+// handleRequestMethod returns true if the request method is among
+// HandledMethods.
+func handleRequestMethod(r *http.Request, cfg *ServerConfig) bool {
+	// For very small slices as these, this should be faster than
+	// a map lookup.
```

Comment:
Someone did the numbers https://boltandnuts.wordpress.com/2017/11/20/go-slice-vs-maps/

Reply:
Be careful. Go is constantly changing.

But this isn't performance critical anyways.

Reply:
Result on go 1.13 mapping string -> bool:

* 1 item - array is faster.
* 2 items - same time.
* 3 items - map is faster.

Reply:
wow